doc/users/prev_whats_new/whats_new_3.8.0.rst

```diff
@@ -327,6 +327,9 @@ This allows users to set the location of the legend in a more flexible and consi
 Mathtext improvements
 =====================
 
+Mathtext is Matplotlib's native TeX-like mathematics parser (:ref:`mathtext`),
```

Comment:
I'm unclear what the statement should communicate. Is it intended as a general description to give context what mathtext is? That'd be a bit odd. We don't have such descriptions for other sections.
Or does it document the improvement in this PR? In that case I suggest

```suggestion
Clarify that Mathtext is Matplotlib's native TeX-like mathematics parser (:ref:`mathtext`),
```

Reply:
"mathtext" is jargon, so it would be nice to define it, at least by a link. None of the code provided helps with the context either, since it could easily apply to usetex labels.

Reply:
Ah, I see, this is an intro paragraph to a set of changes. While we typically don’t have that, I acknowledge  your point on giving context. I’m +/-0 on this. Fundamentally, we can’t explain every concept of the library again in the changelog.
I’d be in favor of only linking, but not explaining. - Possibly even just link the “mathtext” word in the individual improvement messages.

But I withdraw from the discussion now. I’ve added my view and leave it up to you to decide this topic.

Reply:
I don't believe just putting a link in would have helped the OP (why would you click a link for something you think you understand) and folks are often confused by the difference between usetex and mathtext; there is a whole community that _always_ have usetex=True, and probably forget that Matplotlib has mathtext handling.  I don't think it hurts to explicitly remind them here.

Reply:
I changed this sentence to be:

```
Improvements are to Mathtext, Matplotlib's native TeX-like mathematics parser
(see :ref:`mathtext`, not to be confused with Matplotlib using LaTeX directly:
:ref:`usetex`).
```

Reply:
It seems you haven’t pushed yet.

Reply:
ooops....